pkg/tlsutil/tlsutil.go

```diff
@@ -17,6 +17,17 @@ const (
 	Duration365d = time.Hour * 24 * 365
 )
 
+type CertConfig struct {
+	CommonName   string
+	Organization []string
+	AltNames     AltNames
+}
+
+type AltNames struct {
```

Comment:
This should just be unordered list of domain names and IPs. But I can add doc string if that's unclear.